diff -Naur a/sublime_text.desktop b/sublime_text.desktop
--- a/sublime_text.desktop	2021-05-20 05:02:57.000000000 -0400
+++ b/sublime_text.desktop	2021-05-27 16:47:51.663158931 -0400
@@ -15,9 +15,7 @@
 [Desktop Action new-window]
 Name=New Window
 Exec=/opt/sublime_text/sublime_text --launch-or-new-window
-OnlyShowIn=Unity;
 
 [Desktop Action new-file]
 Name=New File
 Exec=/opt/sublime_text/sublime_text --command new_file
-OnlyShowIn=Unity;
